packages/account-v2/src/containers/POOForm/FileUploadField.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { ChangeEvent, MouseEvent, SyntheticEvent, useRef, useState } from 'react';
+import clsx from 'clsx';
+import { Field, FormikErrors, useFormikContext } from 'formik';
+import { StandaloneXmarkRegularIcon } from '@deriv/quill-icons';
+import { Button, Input } from '@deriv-com/ui';
+import { TPaymentMethod, TProofOfOwnershipFormValue } from 'src/types';
+import { compressImageFiles, TFile } from 'src/utils';
+
+type TFileUploadFieldProps = {
+    methodId: number;
+    paymentMethod: TPaymentMethod;
+    subIndex: number;
+};
+
+export const FileUploaderField = ({ methodId, paymentMethod, subIndex }: TFileUploadFieldProps) => {
+    const formik = useFormikContext<TProofOfOwnershipFormValue>();
+    const { errors, setFieldError, setFieldValue, values } = formik;
+    const [showBrowseButton, setShowBrowseButton] = useState(
+        !values[paymentMethod]?.[methodId]?.files?.[subIndex]?.name
+    );
+
+    if (!formik) {
+        throw new Error('FileUploaderField must be used within a Formik component');
+    }
+
+    // Create a reference to the hidden file input element
+    const hiddenInputFieldRef = useRef<HTMLInputElement>(null);
+
+    const preventEventBubble = (e: SyntheticEvent) => {
+        e.nativeEvent.preventDefault();
+        e.nativeEvent.stopPropagation();
```

Comment:
I think while using stopImmediatePropagation we done't need to invoke stopPropagation, do we?

Reply:
Will check this during integration